### ISSUE TITLE
synchronizer: Add a debug feature to call Executor with `NoCounters` flag to true

### ISSUE THREAD

**Issue by joanestebanr** [0xPolygonHermez/zkevm-node#3693]
# Rationale
For a development propuses we need to avoid Counters checks on executor. 
This is a feature forbidden in production environment for that reason the activation is going to be a bit complicated: 
* Activate in the configuration file: 
```
[Synchronizer]
(...)
ExecuteBatchNoCountersFlag = false
```
* Also need a enviornment variable set to allow to use this value: 
```
export ALLOW_EXECUTE_BATCH_NO_COUNTERS_FLAG=1
```
* To prevent to activate this by error, also a invocation parameter is required to allow to set this value:
```
/app/zkevm-node run --allow-execute-batch-no-counters-flag
```
So basically you need the 3 params to be able to enable this feature

Another defensive mechanims is:
- If the chainID is mainnet prevent to change this flag event then rest of conditions are fulfilled

# Implementation

Do you have ideas regarding the implementation of this feature?
Are you willing to implement this feature?
